docs/getting_started/features/transfers.md

```diff
@@ -2,6 +2,23 @@
 
 Transfers provide details about transitions between different travel segments (or legs), enabling trip planners to determine the feasibility of journeys that include transfers.
```

Comment:
I think this section in the GTFS Books is a nice addition to the feature description (I have modified it a bit).

> Specifying transfers doesn't imply passengers can't transfer elsewhere. It just shows whether certain transfers are not possible or require a minimum time to transfer.